docs/user/04-metrics.md

```diff
@@ -860,3 +860,32 @@ spec:
 **Cause**: Gateway cannot receive metrics at the given rate.
 
 **Solution**: Manually scale out the gateway by increasing the number of replicas for the metric gateway. See [Module Configuration and Status](https://kyma-project.io/#/telemetry-manager/user/01-manager?id=module-configuration).
+
+### Scrape metrics from system namespaces
+
+**Symptom**: Metrics from system namespaces are not scraped
```

Comment:
```suggestion
**Symptom**: Metrics from system namespaces are not scraped.
```

Reply:
Is this really the symptom? Is this what the user can observe? 
I would have thought that they can see in the backend that no metrics from system namespaces arrive (similar to the other troubleshooting sections). 
But can they also see what is scraped or not scraped?